### PR TITLE
Reset third_party with build

### PR DESCRIPTION
**Pull request by gauntface**

### CODE REVIEW

**Comment by jeffposnick** [comment]
I'm still not seeing the correctly rendered, syntax-highlighted code blocks. 

Here's what I see when checking out your branch, running `npm install`, `gulp clean`, `gulp serve:prod`, and then visiting http://localhost:3474/get-started/webpack.html:

screen shot 2017-06-12 at 1 34 49 pm

Is there a specific `gulp` task that converts the Markdown into prope code blocks? If so, perhaps it's not listed as a dependency of `gulp serve:prod` and I need to run it explicitly?

File: package.json [hunk 1]
```diff
@@ -48,5 +48,8 @@
     "semver": "^5.3.0",
     "superstatic": "^4.1.0",
     "workbox-build": "0.0.2"
+  },
+  "dependencies": {
+    "prismjs": "^1.6.0"
```
Comment (jeffposnick) [inline package.json]: Can you move this to `devDependency` like the rest of the packages?

**Comment by jeffposnick** [comment]
FWIW, the Markdown to `<pre>` + `<code>` conversion seems to be failing for me, rather than something specific to Prism. (I see the `prism.js` and `prism.css` both loaded now).

I get the text whose source is in the ```javascript code blocks wrapped in `<p>` tags instead of `<pre>` + `<code>`:

screen shot 2017-06-12 at 1 43 01 pm

**Review by jeffposnick — APPROVED** [review]
Things look good after I updated to the latest Jekyll.

**Comment by gauntface** [comment]
Fixes #52

**Pull request merged**